packages/composite-checkout/test/components/checkout-step.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from '../enzyme-adapter';
+
+/**
+ * Internal dependencies
+ */
+import CheckoutStep from '../../src/components/checkout-step';
+
+describe( 'CheckoutStep', function() {
+	const componentWithDefaults = (
+		<CheckoutStep
+			title={ 'Step' }
+			stepNumber={ 1 }
+			isActive={ false }
+			isComplete={ false }
+			stepContent={ <span>empty</span> }
+		/>
+	);
+	it( "Default component is selectable by '.checkout-step'", function() {
+		expect( shallow( componentWithDefaults ).is( '.checkout-step' ) ).toBe( true );
+	} );
+	it( "Default component is not selectable by '.checkout-step--is-active'", function() {
+		expect( shallow( componentWithDefaults ).is( '.checkout-step--is-active' ) ).toBe( false );
+	} );
+	it( "Default component is not selectable by '.checkout-step--is-complete'", function() {
+		expect( shallow( componentWithDefaults ).is( '.checkout-step--is-complete' ) ).toBe( false );
+	} );
+
+	const componentCollapsed = (
+		<CheckoutStep
+			title={ 'Step' }
+			stepNumber={ 1 }
+			isActive={ true }
+			isComplete={ false }
+			stepContent={ <span>empty</span> }
+		/>
+	);
+	it( "Active component is selectable by '.checkout-step'", function() {
+		expect( shallow( componentCollapsed ).is( '.checkout-step' ) ).toBe( true );
+	} );
+	it( "Active component is selectable by '.checkout-step--is-active'", function() {
+		expect( shallow( componentCollapsed ).is( '.checkout-step--is-active' ) ).toBe( true );
+	} );
+
+	const componentWithCustomClass = (
+		<CheckoutStep
+			title={ 'Step' }
+			stepNumber={ 1 }
+			className={ 'custom-class' } // eslint-disable-line wpcalypso/jsx-classname-namespace
+			isActive={ false }
+			isComplete={ false }
+			stepContent={ <span>empty</span> }
+		/>
+	);
+	it( "Component with custom class is selectable by '.checkout-step'", function() {
```

Comment:
This could be `it( 'displays a checkout step when a custom class is used' )` and the following one could be `it( 'uses the custom class when one is set' )`.

Alternatively, we can nest uses of `describe()`:

```
describe( 'CheckoutStep', function() {
  describe( 'with a custom class', function() {
    it( 'still renders the checkout step' );
    it( 'renders the custom class name' );
  } );
} );
```

The latter way is a nice way to organize the test groups too because you can put definitions like `componentWithCustomClass` inside one of the inner `describe` functions so it is scoped just there.